src/components/NextPrevious/__tests__/NextPrevious.test.tsx

```diff
@@ -7,6 +7,7 @@ const routerMock = {
   useRouter: () => {
     return {
       query: { platform: 'react' },
+      asPath: '/react/build-a-backend/auth/manage-user-session',
```

Comment:
Added `asPath` because we're using the `useIsGen1Page()` hook, which looks at `asPath`